src/Microsoft.AspNetCore.Mvc.Core/AuthorizeFilter.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
We could also revisit having a 'filter policy' namespace and using that, but that's more naming by component type